### PR TITLE
Improve metadata for appstream.

### PR DESCRIPTION
**Pull request by darkshram**
Modified appdata file to pass full validation with 'appstream-util validate-strict'.

Before:
```
appstream-util validate-strict io.github.pragha_music_player.metainfo.xml
io.github.pragha_music_player.metainfo.xml: FAILED:
• style-invalid         : Content before <ul> is too short [218], at least 300 characters required
• translations-required  : <name> has no translations
• translations-required  : <summary> has no translations
• translations-required  : <description> has no translations
Validation of files failed
```
After:
```
appstream-util validate-strict data/io.github.pragha_music_player.metainfo.xml 
data/io.github.pragha_music_player.metainfo.xml: OK
```

### CODE REVIEW

**Comment by matiasdelellis** [comment]
Hi @darkshram 
I like the changes, except the translations that must be done by gettext and transifex. :wink: 

Do you want to try? :grimacing:

File: data/io.github.pragha_music_player.metainfo.xml [hunk 1]
```diff
@@ -5,12 +5,30 @@
...
+  <summary xml:lang="zh_CN">管理和听音乐</summary>
   <description>
     <p>
-      Pragha is a Lightweight Music Player for GNU/Linux, based on Gtk, sqlite, and
-      completely written in C, constructed to be fast, light, and simultaneously tries
-      to be complete without obstructing the daily work.
+      Pragha is a Lightweight Music Player for GNU/Linux, based on Gtk3, sqlite,
+      and completely written in C, constructed to be fast, light, and
+      simultaneously tries to be complete without obstructing the daily work.
+      Although it was especifically designed for Xfce4 desktop, it integrates
```
Comment (matiasdelellis) [inline data/io.github.pragha_music_player.metainfo.xml]: I liked this change, but I would say that it is 'thought' for the Xfce desktop, but not specially designed, due it does not fully follow its standards either. :sweat_smile:

File: data/io.github.pragha_music_player.metainfo.xml [hunk 2]
```diff
@@ -5,12 +5,30 @@
...
+  <summary xml:lang="fr">Gérer et écouter de la musique</summary>
+  <summary xml:lang="it">Gestisci e ascolta la musica</summary>
+  <summary xml:lang="ko_KR">음악을 관리하고 들으세요</summary>
+  <summary xml:lang="lt">Tvarkyti ir klausytis muzikos</summary>
+  <summary xml:lang="nl">Beheer en luister naar muziek</summary>
+  <summary xml:lang="pt">Gestão e reprodução de músicas</summary>
+  <summary xml:lang="ru">Слушайте и управляйте музыкой</summary>
+  <summary xml:lang="tr">Müziği yönetin ve dinleyin</summary>
+  <summary xml:lang="uk">Слухайте та керуйте музикою</summary>
+  <summary xml:lang="zh_CN">管理和听音乐</summary>
```
Comment (matiasdelellis) [inline data/io.github.pragha_music_player.metainfo.xml]: Translations must be done automatically by gettex and transifex.

**Comment by darkshram** [comment]
I'll try in a few days. I have a little tight schedule. io.github.pragha_music_player.metainfo.xml will need to be renamed to io.github.pragha_music_player.metainfo.xml.in and to modify Makefile to do the rest. `<name>, <summary> and <description>` tags just need to be replaced with `<_name>, <_summary> and <_description>` and obviously have an entry in the pot and po files.  For the time being, just remove the lines with xml:lang and modify whatever yo need, so pragha's appdata will look awesome in app stores. Appdata will validate Ok with 'appstream-util validate' instead of 'appstream-util validate-strict' (only validate-strict requires `<name>, <summary> and <description>` to have translations).

**Comment by matiasdelellis** [comment]
Hi @darkshram 
Wow, Thank you for answering so quickly. I'm doing the changes form translation with gettex. Wait some minutes. :wink:

**Comment by matiasdelellis** [comment]
Well, Finally I add all your changes in the last master commit. Please when you can, look at it.. :wink: 

```
[matias@ideapad pragha]$ appstream-util validate data/io.github.pragha_music_player.metainfo.xml
data/io.github.pragha_music_player.metainfo.xml: OK
```

But in strict mode fail due intool remove the copyright comment.. :confused: 

```
[matias@ideapad pragha]$ appstream-util validate-strict data/io.github.pragha_music_player.metainfo.xml
data/io.github.pragha_music_player.metainfo.xml: FAILED:
• value-missing         : <!-- Copyright [year] [name] --> is not present
Validation of files failed
```


p.s: It's a shame that will not be properly registered -As git commit authored- that these are your changes..  Sorry.. :disappointed:

**Comment by darkshram** [comment]
> 
> p.s: It's a shame that will not be properly registered -As git commit authored- that these are your changes.. Sorry.. disappointed

Don't worry. The goal was to improve visibility of pragha in app stores.

Keywords may be localized too. I would recommend to add 'player'.

```
  <keywords>
    <_keyword>audio</keyword>
    <_keyword>multimedia</keyword>
    <_keyword>player</keyword>
  </keywords>
```
If you add 'Keywords' key to the desktop file too, you may reuse localizations.

```
[Desktop Entry]
Version=1.0

Type=Application

Name=Pragha
_GenericName=Music Player
_Comment=Manage and listen your music
Icon=pragha

Categories=GTK;AudioVideo;Player;
_Keywords=audio;multimedia;player;

Exec=pragha %F
TryExec=pragha
Terminal=false
StartupNotify=true

MimeType=application/x-ape;audio/ape;audio/x-ape;audio/x-m4a;video/x-ms-asf;audio/x-ms-wma;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/mpeg3;audio/mp3;application/ogg;application/x-ogg;audio/vorbis;audio/x-vorbis;audio/ogg;audio/x-ogg;audio/x-flac;application/x-flac;audio/flac;audio/x-wav;audio/mpegurl;audio/x-mpegurl;audio/x-scpls;application/xspf+xml;audio/x-ms-wax;

```

**Comment by matiasdelellis** [comment]
Done.. :grimacing: 
Thanks!.

**Pull request closed without merging**